backend/src/Repository/Package/PackageEntityRepository.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Repository\Package;
+
+use App\Entity\PackageEntity;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+use App\Constant\Package\PackageConstant;
+
+/**
+ * @method PackageEntity|null find($id, $lockMode = null, $lockVersion = null)
+ * @method PackageEntity|null findOneBy(array $criteria, array $orderBy = null)
+ * @method PackageEntity[]    findAll()
+ * @method PackageEntity[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class PackageEntityRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, PackageEntity::class);
+    }
+
+    public function getActivePackages(): mixed
```

Comment:
Why "mixed" is the returned type instead of "array"?

Reply:
changes must apply to all other functions.